formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -766,7 +766,8 @@ echoTheEchoer(
       o,
       p
     )
-  ) => (
+  ) =>
```

Comment:
Collateral damage of Easy_format's limitations, will be addressed in the future.